### PR TITLE
Move from Travis to GA

### PR DESCRIPTION
**Pull request by abhinayagarwal**
Adds workflows to:

* Build PR and deploy snapshots on every commit to master
* Release to Maven Central when tags are pushed

All secrets have been added to the repository

### CODE REVIEW

**Review by erwin1 — APPROVED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+          server-username: MAVEN_USERNAME
+          server-password: MAVEN_CENTRAL_TOKEN
+
+      - name: Publish Snapshots
+        run: |
+          # Find version
+          ver=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
+          if [[ $ver == *"SNAPSHOT"* ]]
+          then
+              mvn deploy -DskipTests=true
```
Comment (tiainen) [inline .github/workflows/build.yml:31]: We should not skip running tests during development?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+name: Build
```
Comment (tiainen) [inline .github/workflows/build.yml:1]: I noticed that `master.yml` has been removed. Do we no longer need to run the `release-drafter` action?

Comment (abhinayagarwal) [reply .github/workflows/build.yml:1]: Good catch, fixed!

File: .github/workflows/build.yml (hunk 1 above)
Comment (abhinayagarwal) [reply .github/workflows/build.yml]: Fixed

**Review by tiainen — APPROVED** [review]

**Pull request merged**